JSDemos/Demos/DataGrid/CustomEditors/Angular/app/app.component.ts

```diff
@@ -67,14 +67,18 @@ export class AppComponent {
     if (target === 'search' && typeof (filterValue) === 'string') {
       return [(this as any).dataField, 'contains', filterValue];
     }
-    return function (data) {
-      return (data.AssignedEmployee || []).indexOf(filterValue) !== -1;
+    return function (rowData) {
+      return (rowData.AssignedEmployee || []).indexOf(filterValue) !== -1;
```

Comment:
I cant left my comments to unchanged line, so posting here:

on line 68 remove please `(this as any)`, it's better to write type for `this` in callback arguments

Reply:
Let's do typings for Angular demos is a separate PR, because in Angular demos there are lot of places where we need to add types.